src/commands/environment/backupRestore/backup.ts

```diff
@@ -49,6 +49,10 @@ export const register: RegisterCommand = yargs =>
           alias: "x",
           choices: backupEntityChoices,
           conflicts: "include",
+        })
+        .option("managementApiUrl", {
+          type: "string",
+          describe: "URL of the Kontent.ai Management API.",
```

Comment:
Nitpick: what about "Custom URL for Kontent.ai MAPI calls"?

Reply:
also migration toolkit commands also has option for baseUrl, maybe add it also there?